Makefile

```diff
@@ -6,15 +6,18 @@ SHORT_GIT_HASH := $(shell git rev-parse --short HEAD)
 NGC_REGISTRY := nvcr.io/isv-ngc-partner/determined
 export DOCKERHUB_REGISTRY := determinedai
 
-CPU_PREFIX := environments:py-3.7-
+CPU_PREFIX := environments:py-3.8-
+CPU_PREFIX_37 := environments:py-3.7-
 CPU_SUFFIX := -cpu
 CUDA_102_PREFIX := environments:cuda-10.2-
```

Comment:
Thinking aloud: it would make the name even longer, but it's probably useful to include the Python version in the GPU images one day. Don't know why we never have, we just replace it with CUDA. Let's get this merged, regardless.

Reply:
yep I thought the same while updating this.